app/(candidate)/dashboard/website/editor/components/WebsiteEditFlow.js

```diff
@@ -161,12 +161,17 @@ export default function WebsiteEditFlow() {
     }))
   }
 
-  function handleAddressChange(value) {
+  function handleAddressChange(value, place) {
+    console.log('place', place)
```

Comment:
File got renamed but console log is still there: https://github.com/thegoodparty/gp-webapp/pull/729#discussion_r2219917826